### ISSUE TITLE
test label

### ISSUE THREAD

**Issue by SunChengGong** [PRBonn/semantic-kitti-api#166]
I would like to know how to get the labels for the test set so that I can evaluate my model

**Comment by jbehley** [comment]
Please note that we do not release the test labels, and you have to submit your predictions to the corresponding Codalab Challenges: https://semantic-kitti.org/tasks.html

For each of the different tasks, we have a separate Codalab competition. Note that we restrict the number of (valid) submissions to 10 and only accept email addresses from official accounts (university, companies, etc.) to prevent multiple registrations. Ablations and hyperparameter tuning should be only conducted on the validation set.

**Comment by jbehley** [comment]
I guess also this issues has been clarified. If you have further problems, then open a new issue or re-open this issue.
